src/workerd/io/worker.c++

```diff
@@ -2816,8 +2813,8 @@ kj::Maybe<ActorCacheInterface&> Worker::Actor::getPersistent() {
 
 kj::Maybe<jsg::Ref<api::DurableObjectStorage>>
     Worker::Actor::makeStorageForSwSyntax(Worker::Lock& lock) {
-  return impl->actorCache.map([&](ActorCacheInterface& cache) {
-    return impl->makeStorage(lock, *worker->getIsolate().apiIsolate, cache);
+  return impl->actorCache.map([&](kj::Own<ActorCacheInterface>& cache) {
```

Comment:
no action: It would be nice if we could augment `kj::Maybe<>::map()` to take lambdas for ref or raw pointer. I understand that it's harder to do right before requires expressions and constexpr if though.